subscription-offer-prompt/test/lionel.spec.js

```diff
@@ -87,12 +105,112 @@ describe('"Lionel Slider" Subscription Offer Prompt', () => {
 
 	it('should not show if last shown within 30 days', () => {
 		localStorage.set('last-closed', Date.now() + (1000 * 60 * 60 * 24 * 29));
-		return init().then(popup => should.not.exist(popup));
+		return init(flags).then(popup => should.not.exist(popup));
 	});
 
 	it('should not show barrier page has not been visited in this session', () => {
 		sessionStorage.unset('last-seen');
-		return init().then(popup => should.not.exist(popup));
+		return init(flags).then(popup => should.not.exist(popup));
+	});
+
+});
+
+describe('"Lionel Slider" Subscription Offer Prompt - USA', () => {
```

Comment:
The way describe block nesting works means that you could define the stubs and fixtures that won't change at the top of the file in the top level describe block.

That way you only have to define the stubs you want to change (country) in the nested blocks. And not have to redefine additional stubs each time.